lib/compress.js

```diff
@@ -138,6 +139,9 @@ function Compressor(options, false_by_default) {
         funcs: toplevel,
         vars: toplevel
     };
+    if (this.options['module']) {
+        this.directives['use strict'] = true;
```

Comment:
If `compress` `module` is enabled, then `compress` `toplevel` can also be enabled.